src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs

```diff
@@ -90,5 +91,13 @@ public DocumentDBAttribute(string databaseName, string collectionName)
         /// collection.
         /// </summary>
         public int CollectionThroughput { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// When specified on an input binding using an <see cref="IEnumerable{T}"/>, defines the query to run against the collection. 
```

Comment:
The plan was that if you supply `SqlQuery`, you must use an IEnumerable -- I'm adding tests to make sure that's the case. As well as make sure if you specify both `Id` and `SqlQuery`, we throw, as that is unclear.

So if you expect a single result from a query, you'll need to pull it from the returned enumerable.